grpc/src/main/java/io/stargate/grpc/service/BatchHandler.java

```diff
@@ -93,6 +94,8 @@ protected void validate() throws Exception {
 
   @Override
   protected CompletionStage<ResponseAndTraceId> buildResponse(Result result) {
+    System.out.println(
```

Comment:
There are bunch of `System.out.println` calls in this PR, I guess that this will be removed?